src/gluonnlp/vocab/bert.py

```diff
@@ -70,25 +78,23 @@ class BERTVocab(Vocab):
         A token used to separate sentence pairs for BERT.
     cls_token : hashable object or None, default '[CLS]'
         Classification symbol for BERT.
-    reserved_tokens : list of hashable objects or None, default [mask_token, sep_token, cls_token],
-        A list of reserved tokens (excluding `unknown_token`) that will always be indexed, such as
-        special symbols representing padding, beginning of sentence, and end of sentence. It cannot
-        contain `unknown_token` or duplicate reserved tokens. Keys of `counter`, `unknown_token`,
-        and values of `reserved_tokens` must be of the same hashable type. Examples: str, int, and
-        tuple.
+    reserved_tokens : list of hashable objects or None, default None
+        A list specifying additional tokens to be added to the vocabulary.
+        `reserved_tokens` cannot contain `unknown_token` or duplicate reserved
+        tokens.
+        Keys of `counter`, `unknown_token`, and values of `reserved_tokens`
+        must be of the same hashable type. Examples of hashable types are str,
+        int, and tuple.
+    token_to_idx : dict mapping tokens (hashable objects) to int or None, default None
+        Optionally specifies the indices of tokens to be used by the
+        vocabulary. Each token in `token_to_index` must be part of the Vocab
+        and each index can only be associated with a single token.
+        `token_to_idx` is not required to contain a mapping for all tokens. For
+        example, it is valid to only set the `unknown_token` index to 10 (instead
+        of the default of 0) with `token_to_idx = {'<unk>': 10}`.
 
     Attributes
     ----------
-    UNKNOWN_TOKEN : '[UNK]'
-        Static attribute of BERTVocab, unknown token for BERT.
-    PADDING_TOKEN : '[PAD]'
-        Static attribute of BERTVocab, padding token for BERT.
-    MASK_TOKEN : '[MASK]'
-        Static attribute of BERTVocab, mask token for BERT.
-    SEP_TOKEN : '[SEP]'
-        Static attribute of BERTVocab, a token used to separate sentence pairs for BERT.
-    CLS_TOKEN : '[CLS]'
-        Static attribute of BERTVocab, Classification symbol for BERT.
```

Comment:
Should these attributes still be documented? @eric-haibin-lin what do you think?

Reply:
They are removed, so as long as they are not moved back, they shouldn't be documented. We can consider moving them back if you feel it's better